lib/epochs.js

```diff
@@ -544,13 +545,18 @@ function buildPreferredEpoch(reduce, cb) {
   var preferredEpoch
   if (tips.length === 1) preferredEpoch = tips[0]
   else if (tips.length === 2) {
-    if (isSameSet(...tips.map((t) => t.members))) {
+    const [members1, members2] = tips.map((t) => t.members)
+    if (isSameSet(members1, members2)) {
       preferredEpoch = tieBreak(tips)
-    } else {
-      return cb(
-        Error('Tips do not have same membership - case not handled yet')
-      )
+    } else if (isSubsetOf(members1, members2)) {
+      preferredEpoch = tips[0]
+    } else if (
+      isSubsetOf(members2, members1)
+    ) {
+      preferredEpoch = tips[1]
     }
```

Comment:
could be nice with some comments here or?

Reply:
done

Reply:
mm where?